src/api/stats/getAmmPrices.ts

```diff
@@ -675,9 +676,16 @@ const updateAmmPrices = async () => {
 
     const linearPoolPrice = ammPrices.then(async ({ poolPrices, tokenPrices, _ }) => {
       const jbrlTokenPrice = await fetchJbrlPrice(tokenPrices);
+      const yvWFTMPrice = await fetchyvWFTMPrice(tokenPrices);
```

Comment:
any other way to calculate it?
it is not really good to add random single tokens here without multicall and error handling.
like jBRL shouldn't be here as well.
ideally use uniV2 LP and calc via `oldLpPools` 
or make `fetchErc4626Prices` which would be similar to fetchMooPrices or fetchVaultPrices.
or combine it with jBRL and make multicall multichain `fetchBalancerPrices`, so future balancer tokens can be just declared there.